docs/user/types/full_product_name/product_identification_helper/purl-usage.en.md

```diff
@@ -1,21 +1,23 @@
 # Purl - Usage
 
-* A purl is a URL composed of seven components which are separated by a specific character for unambiguous parsing:
+A purl is a URL composed of seven components which are separated by a specific character for unambiguous parsing:
 
-  ```text
-  scheme:type/namespace/name@version?qualifiers#subpath
-  ```
+```text
+scheme:type/namespace/name@version?qualifiers#subpath
+```
 
-  The defintion for each components is:
+The defintion for each components is:
 
-  * `scheme`: this is the URL scheme with the constant value of "pkg". One of the primary reason for this single scheme is to facilitate the future official registration of the "pkg" scheme for package URLs. Required.
-  * `type`: the package "type" or package "protocol" such as maven, npm, nuget, gem, pypi, etc. Required.
-  * `namespace`: some name prefix such as a Maven groupid, a Docker image owner, a GitHub user or organization. Optional and type-specific.
-  * `name`: the name of the package. Required.
-  * `version`: the version of the package. Optional.
-  * `qualifiers`: extra qualifying data for a package such as an OS, architecture, a distro, etc. Optional and type-specific.
-  * `subpath`: extra subpath within a package, relative to the package root. Optional.
+* `scheme`: this is the URL scheme with the constant value of "pkg". One of the primary reason for this single scheme is to facilitate the future official registration of the "pkg" scheme for package URLs. Required.
+* `type`: the package "type" or package "protocol" such as maven, npm, nuget, gem, pypi, etc. Required.
+* `namespace`: some name prefix such as a Maven groupid, a Docker image owner, a GitHub user or organization. Optional and type-specific.
+* `name`: the name of the package. Required.
+* `version`: the version of the package. Optional.
+* `qualifiers`: extra qualifying data for a package such as an OS, architecture, a distro, etc. Optional and type-specific.
+* `subpath`: extra subpath within a package, relative to the package root. Optional.
 
-  Components are designed such that they form a hierarchy from the most significant component on the left to the least significant component on the right.
+Components are designed such that they form a hierarchy from the most significant component on the left to the least significant component on the right.
 
-* A purl must NOT contain a URL Authority i.e. there is no support for username, password, host and port components. A namespace segment may sometimes look like a host but its interpretation is specific to a type.
+A purl must NOT contain a URL Authority i.e. there is no support for username, password, host and port components. A namespace segment may sometimes look like a host but its interpretation is specific to a type.
```

Comment:
Please also clarify that there is no `//` after `pkg:` as:

> Since a purl never contains a URL Authority, its scheme must not be suffixed with double slash as in 'pkg://' and should use instead 'pkg:'. Otherwise this would be an invalid URI per rfc3986 at https://tools.ietf.org/html/rfc3986#section-3.3: